tests/qam-updinstall/update_install.pm

```diff
@@ -206,6 +206,10 @@ sub run {
     # Extract module name from repo url.
     my @modules = split(/,/, $repos);
     foreach (@modules) {
+        # substitue SLES_SAP for LTSS repo at this point is SAP ESPOS
+        # only availabe for 15-SP2 since SLE-Product-SLES_SAP_15-SP2 channel is empty
+        $_ =~ s/SAP_(\d+(-SP\d)?)/$1-LTSS/ if is_sle('15+');
+
```

Comment:
LGTM.
Only one concern: do we need a VR for 15SP3/4/5/6 ?

Reply:
I think no need to run VR because no issue happened with this code on 15+ before.
the code deleted before:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20616/files

Reply:
Sure. Thanks.